sda-commons-web-autoconfigure/src/main/java/org/sdase/commons/spring/boot/web/monitoring/SdaTracingConfiguration.java

```diff
@@ -7,9 +7,22 @@
  */
 package org.sdase.commons.spring.boot.web.monitoring;
 
+import io.opentelemetry.api.OpenTelemetry;
 import org.springframework.boot.autoconfigure.AutoConfiguration;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.PropertySource;
 
 @PropertySource("classpath:/org/sdase/commons/spring/boot/web/monitoring/tracing.properties")
 @AutoConfiguration
-public class SdaTracingConfiguration {}
+public class SdaTracingConfiguration {
+
+  @Bean
+  @ConditionalOnProperty(
+      value = "management.otlp.disable",
+      havingValue = "true",
+      matchIfMissing = true)
```

Comment:
If `matchIfMissing` is `true`, a would expect that the default is the `noop` tracer. But shouldn't we have tracing enabled by default?

Reply:
I agree, but for the tests it must be disabled by default. I added to the test system properties [here](https://github.com/SDA-SE/sda-spring-boot-commons/pull/301/files#diff-943d09cac58ce340d7fb07e57228f6feaf0d8513e7cf277cd02de523284ce0f1R41)

Reply:
Does that apply for users of the library as well?
Would it make sense to disable it in `src/test/resources/application.properties` instead of the gradle file?